### PR TITLE
Change "video" to "medium", as the word should cover both audio and video

### PR DESCRIPTION
**Pull request by Stadly**

### CODE REVIEW

File: content/tutorial/02-advanced-svelte/05-bindings/03-media-elements/README.md [hunk 1]
```diff
@@ -65,7 +65,7 @@ When the track ends, be kind — rewind:
 
 The complete set of bindings for `<audio>` and `<video>` is as follows — seven _readonly_ bindings...
 
-- `duration` (readonly) — the total duration of the video, in seconds
+- `duration` (readonly) — the total duration of the medium, in seconds
```
Comment (Rich-Harris) [inline content/tutorial/02-advanced-svelte/05-bindings/03-media-elements/README.md:68]:
```suggestion
- `duration` (readonly) — the total duration of the media, in seconds
```

File: content/tutorial/02-advanced-svelte/05-bindings/03-media-elements/README.md [hunk 2]
```diff
@@ -75,8 +75,8 @@ The complete set of bindings for `<audio>` and `<video>` is as follows — seven
 
 ...and five _two-way_ bindings:
 
-- `currentTime` — the current point in the video, in seconds
-- `playbackRate` — how fast to play the video, where `1` is 'normal'
+- `currentTime` — the current point in the medium, in seconds
+- `playbackRate` — how fast to play the medium, where `1` is 'normal'
```
Comment (Rich-Harris) [inline content/tutorial/02-advanced-svelte/05-bindings/03-media-elements/README.md:79]:
```suggestion
- `currentTime` — the current point in the media, in seconds
- `playbackRate` — how fast to play the media, where `1` is 'normal'
```

**Review by Rich-Harris — COMMENTED** [review]
thank you! though it's 'media', not 'medium'

File: content/tutorial/02-advanced-svelte/05-bindings/03-media-elements/README.md [hunk 3]
```diff
@@ -65,7 +65,7 @@ When the track ends, be kind — rewind:
 
 The complete set of bindings for `<audio>` and `<video>` is as follows — seven _readonly_ bindings...
 
-- `duration` (readonly) — the total duration of the video, in seconds
+- `duration` (readonly) — the total duration of the media, in seconds
```
Comment (Rich-Harris) [inline content/tutorial/02-advanced-svelte/05-bindings/03-media-elements/README.md:68]: actually, i really don't like how this reads. gonna just rework it

```suggestion
- `duration` (readonly) — the total duration, in seconds
```

File: content/tutorial/02-advanced-svelte/05-bindings/03-media-elements/README.md [hunk 4]
```diff
@@ -75,8 +75,8 @@ The complete set of bindings for `<audio>` and `<video>` is as follows — seven
 
 ...and five _two-way_ bindings:
 
-- `currentTime` — the current point in the video, in seconds
-- `playbackRate` — how fast to play the video, where `1` is 'normal'
+- `currentTime` — the current point in the media, in seconds
+- `playbackRate` — how fast to play the media, where `1` is 'normal'
```
Comment (Rich-Harris) [inline content/tutorial/02-advanced-svelte/05-bindings/03-media-elements/README.md:79]:
```suggestion
- `currentTime` — the current position of the playhead, in seconds
- `playbackRate` — speed up or slow down (`1` is 'normal')
```

**Review by Rich-Harris — APPROVED** [review]

**Comment by Stadly** [comment]
Thanks. I was wondering myself, so I asked [ChatGPT](https://chat.openai.com/share/4e585279-c114-4688-82c7-dbe4967f98c8) 😅 But I think you're right :)

**Pull request merged**